src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ClientConfigPaths.cs

```diff
@@ -47,8 +48,11 @@ private ClientConfigPaths(string exePath, bool includeUserConfig)
             {
                 // Exe path wasn't specified, get it from the entry assembly
                 exeAssembly = Assembly.GetEntryAssembly();
+                var exeName = exeAssembly?.ManifestModule.Name;
```

Comment:
Nit: The [coding conventions](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md) say to use explicit type instead of `var` here.